components/ContentTemplates/ChartsTemplate.tsx

```diff
@@ -1,5 +1,101 @@
-import { WorkInProgress } from "../WorkInProgress/WorkInProgress"
+import { CodeBlock } from "../CodeBlock/CodeBlock"
+import { NavPage } from "../NavPage/NavPage"
+import { PageUpdated } from "../PageUpdated/PageUpdated"
+import { TemplateSection } from "../TemplateSection/TemplateSection"
+import { chartPageNavigation } from "../../data/pageNavigationLists"
```

Comment:
Please add the changes to chartPageNavigation in the pageNavigationLists file to this pull request, otherwise the page won't load.

Reply:
How do I do that @EmmaDawsonDev?

Reply:
If you go to the file named pageNavigationLists.ts in the data folder you will see that for every page that already has content there is an array like this:

```
export const audioPageNavigation: IPageNavigationItem[] = [
	{ linkName: "Introduction", href: "#introduction" },
	{ linkName: "Transcripts", href: "#transcripts" },
	{ linkName: "Captions", href: "#captions" },
	{ linkName: "How To Add Closed Captions To Your Video?", href: "#howToAdd" },
	{ linkName: "Sign Language", href: "#signLanguage" },
	{ linkName: "Accessible media players", href: "#accessibleMediaPlayers" },
	{ linkName: "WCAG Criteria", href: "#WCAGCriteria" },
	{ linkName: "Other Resources", href: "#otherResources" },
]
```

These relate to each section on the page and are used in the in-page navigation. The linkName should match the templateSection title and the href should match the templateSection sectionName.